code/game/machinery/doors/door.dm

```diff
@@ -278,7 +278,7 @@
 	. = ..()
 	if(.)
 		return
-	if(modifiers[RIGHT_CLICK])
+	if(islist(modifiers) && modifiers[RIGHT_CLICK])
```

Comment:
this is why god made `modifiers?[RIGHT_CLICK]` (or LAZYACCESS(modifiers, RIGHT_CLICK)